msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -197,7 +197,7 @@ Add a [webApplicationInfo](../../resources/schema/manifest-schema.md#webapplicat
 |---|---|---|
 |`id` |String |Your Azure AD app ID. For more information, see [register your app in the Azure AD portal](resource-specific-consent.md#register-your-app-with-microsoft-identity-platform-using-the-azure-ad-portal).|
 |`resource`|String| This field has no operation in RSC, but must be added and have a value to avoid an error response; any string will do.|
-|`applicationPermissions`|Array of strings|RSC permissions for  your app. For more information, see [resource-specific permissions](resource-specific-consent.md#resource-specific-permissions).|
+|`applicationPermissions`|Array of strings|RSC permissions for  your app. For more information, see [Supported RSC permissions](resource-specific-consent.md#supported-rsc-permissions).|
```

Comment:
We should update RSC definition on this page too. 

Maybe call it - Test application RSC permissions

Reply:
Updated